tests/integration/tests/conftest.py

```diff
@@ -65,13 +67,32 @@ def node_count(request) -> int:
 
 
 @pytest.fixture(scope="function")
-def disable_k8s_bootstrapping(request) -> int:
+def disable_k8s_bootstrapping(request) -> bool:
     return bool(request.node.get_closest_marker("disable_k8s_bootstrapping"))
 
 
+@pytest.fixture(scope="function")
+def bootstrap_config(request) -> Path | None:
```

Comment:
I didn't notice that we are using `py3.8`. Instead, we can use:
```suggestion
def bootstrap_config(request) -> Union[Path, None]:
```